3drepo/3drepo.io
# ISSUE #3725 - V5 Visual Settings Facelift

This fixes #3725

#### Description
Adds the visual settings modal accessible from the user dropdown menu

#### Test cases
- Check v4 modal is unaffected
- View warning message when enabling model caching
- View error tooltip when entering an invalid value
